recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-core/images/core-image-thin-initramfs.bbappend

```diff
@@ -25,6 +25,8 @@ python __anonymous () {
         d.appendVar("IMAGE_INSTALL", " domf domf-run domf-install-artifacts")
     if "domr" in guests :
         d.appendVar("IMAGE_INSTALL", " domr domr-run domr-install-artifacts")
+    if "domu" in guests :
```

Comment:
The same. Actually, we can have `guest_depends` in some global variable, to eliminate  this copy-pasted code.

Reply:
this is a refactoring which needs to be done later